_studio/mfx_lib/shared/include/mfx_brc_common.h

```diff
@@ -100,7 +102,7 @@ class AVGBitrate
         maxWinBitsLim = MFX_MIN(maxWinBitsLim + recode*GetStep() / 2, m_maxWinBits);
 
         mfxU32 maxFrameSize = winBits >= m_maxWinBitsLim ?
-            m_maxWinBits - winBits :
+            (mfxU32)(IPP_MAX((mfxI32)m_maxWinBits - (mfxI32)winBits, 1)) :
```

Comment:
please, don't introduce new usages of MAX macro. I hope to merge #925 and changes like this will require painfull rebase...

Use ```std::max``` instead